README.adoc

```diff
@@ -105,7 +105,7 @@ Finally, there are rules checking the proper usage of third-party library code:
 
 === Naming Convention Rules
 
-With the next update, we will add rules checking for adherence to our naming convention rules:
+With our 3.2.0 release, we added rules checking for adherence to our naming convention rules:
```

Comment:
on the long run, it is better to remove historical information in README and keep that only in CHANGELOG. Untill these rules are not yet available, it is fine to keep this...